### PR TITLE
Update device-images.md

### PR DESCRIPTION
**Pull request by kmsft**
- clarified scope of M365 App image settings
- actions taken by W365

### CODE REVIEW

File: windows-365/enterprise/device-images.md [hunk 1]
```diff
@@ -66,6 +66,7 @@ There are two sets of images available to choose from across the different versi
   - Microsoft Teams (Teams).
   - Microsoft Edge settings like sleeping tabs, startup boost, and first time optimizations based on Azure AD and synchronization.
   - Microsoft Outlook first-time configuration settings (auto log-on based on Azure AD profile, support for other profiles).
+  - Settings listed under **Images with OS optimizations**
```
Comment (JohanFreelancer9) [inline windows-365/enterprise/device-images.md:69]:
```suggestion
  - Settings listed under **Images with OS optimizations**.
```

File: windows-365/enterprise/device-images.md [hunk 2]
```diff
@@ -103,8 +104,9 @@ When you upload a custom device image, Windows 365:
 2. Runs the following validation checks on the image:
     1. Verifies all the Windows 365 image requirements are met.
     2. Deploys a virtual machine and makes sure that the images can be booted and provisioned as a Cloud PC.
-3. If you have a Hybrid Azure AD Join connection, Windows 365 replicates the image across all Azure regions where you have an Azure network connection.
-4. If you have an Azure AD Join connection, Windows 365 replicates the image to the provisioned region during provisioning.
+3. If you have
+    - a Hybrid Azure AD Join connection, Windows 365 replicates the image across all Azure regions where you have an Azure network connection.
+    - an Azure AD Join connection, Windows 365 replicates the image to the provisioned region during provisioning.
```
Comment (JohanFreelancer9) [inline windows-365/enterprise/device-images.md:109]:
```suggestion
3. If you have a Hybrid Azure AD Join connection, Windows 365 replicates the image across all Azure regions where you have an Azure network connection.
4. If you have an Azure AD Join connection, Windows 365 replicates the image to the provisioned region during provisioning.
```

Comment (JohanFreelancer9) [reply windows-365/enterprise/device-images.md:109]: I do not understand why the original formatting and text need to be changed. Could you please clarify?

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @kmsft, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by ErikjeMS** [comment]
#sign-off

**Pull request closed without merging**